package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -15,7 +15,7 @@ ADD_LOCAL_HOSTNAME=1
 CONFIGFILE="/var/etc/dnsmasq.conf"
 HOSTFILE="/tmp/hosts/dhcp"
 TRUSTANCHORSFILE="/usr/share/dnsmasq/trust-anchors.conf"
-TIMESTAMPFILE="/etc/dnsmasq.time"
+TIMEVALIDFILE=/var/state/dnsmasqsec
```

Comment:
I've just checked one of my OpenWRT CC router, /var/state is world writable
Can someone double check on a recent LEDE image